src/index.ts

```diff
@@ -35,7 +36,14 @@ export default createBuilder(
       // the port of the Angular Live Development Server is passed to the Electron window
       // so that it knows exactly which URL should load
       const port = result.port as string;
-      spawn(electronPath, [appPath, port], { shell: true });
+      // the devTools parameter is passed to the Electron window so that it knows whether
+      // to open the dev tools or not
+      const devTools =
+        options.devTools != null && `${options.devTools}` !== 'false';
```

Comment:
Can you please write some comments about why we make this particular complex check?